### DISCUSSION TITLE
How we can add or create game in Resource?

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-coKreat/Community#62] [Q&A]
I would need to understand how this part works.
How to add a game in Resource?
![image](https://user-images.githubusercontent.com/109504832/227851293-ff7bbfbc-111b-4780-b6f3-6ceeaa4f4376.png)
![image](https://user-images.githubusercontent.com/109504832/227851315-1d2f2f6e-4c08-43e7-aff1-daf9158c1e83.png)

**Comment by mmanimegalai** [comment]
@sachinb80 Currently there are two ways to create games
1. Create a game using html5 and upload it using the "Upload content" option from workspace
2. Create game as an activity plugin using ECML specification and add it as a Resource

cc @vinukumar-vs

**Reply by sachinb80** [reply]
@mmanimegalai thanks for the reply.
Actually, I am interested in **create game as an activity plugin using ECML specification and add it as a Resource**
Can you please guide me how to do it?

**Comment by mmanimegalai** [comment]
Tagging @pallakartheekreddy

**Reply by sachinb80** [reply]
@pallakartheekreddy Hi, Can you please help with this?

**Reply by sachinb80** [reply]
Hi @pallakartheekreddy

**Reply by pallakartheekreddy** [reply]
@sachinb80 I have added my comments in below thread please check
https://github.com/orgs/Sunbird-Knowlg/discussions/115
